provider/cmd/pulumi-resource-nuage/nuage_provider/container_function.py

```diff
@@ -168,12 +202,47 @@ def __init__(self, name: str, args: ContainerFunctionArgs, props: Optional[dict]
             )
         )
         
+        # FIXME: Thhis doesn't work because pulumi Command doesn't have option to run on update right now 
+        # As it work only on create, it keeps having the same tag on updates
```

Comment:
https://www.pulumi.com/registry/packages/command/api-docs/local/command/#update_python

Wouldn't that work on updates ??